modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMContentLocalServiceImpl.java

```diff
@@ -22,14 +22,11 @@
 /**
  * @author     Brian Wing Shun Chan
  * @author     Eduardo Lundgren
- * @deprecated As of Cavanaugh (7.4.x), replaced by {@link
- *             DDMFieldLocalServiceImpl}
  */
 @Component(
 	property = "model.class.name=com.liferay.dynamic.data.mapping.model.DDMContent",
 	service = AopService.class
 )
-@Deprecated
```

Comment:
@icgsbr please also run `buildService` after this commit, I believe the `DDMContentLocalServiceBaseImpl` will be updated.

Reply:
done